voluncheer/map/views.py

```diff
@@ -27,16 +27,17 @@ def map(request):
     opportunities = []
     for opportunity in Opportunity.objects.all():
         if opportunity.is_published and not opportunity.is_archived:
-            opportunities.append(
-                {
-                    "latitude": opportunity.latitude,
-                    "longitude": opportunity.longitude,
-                    "title": opportunity.title,
-                    "address": opportunity.address_1,
-                    "type": opportunity.category.name,
-                    "name": opportunity.organization.name,
-                }
-            )
+            if opportunity.latitude and opportunity.longitude:
```

Comment:
Instead of nesting even further let's move this logic out into a single function, something like

```
def should_show_opportunity(opp):
    return opp.is_published and not opp.is_archivhed and opp.latitude and opp.longitude
```

WDYT?